File: app/src/main/java/ru/nikshlykov/englishwordsapp/ui/study/ChooseFromFourVariantsModeFragment.java
package ru.nikshlykov.englishwordsapp.ui.study;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import ru.nikshlykov.englishwordsapp.R;

public class ChooseFromFourVariantsModeFragment extends Fragment {


    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_choose_from_four_variants_mode, null);
    }


}

File: app/src/main/java/ru/nikshlykov/englishwordsapp/ui/fragments/ChooseFromFourVariantsModeFragment.kt
package ru.nikshlykov.englishwordsapp.ui.fragments

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import ru.nikshlykov.englishwordsapp.R

class ChooseFromFourVariantsModeFragment : Fragment() {
  override fun onCreateView(
    inflater: LayoutInflater,
    container: ViewGroup?,
    savedInstanceState: Bundle?
  ): View? {
    return inflater.inflate(R.layout.fragment_choose_from_four_mode, null)
  }
}